src/modules/LanItems.rb

```diff
@@ -2460,6 +2391,173 @@ def drop_hosts(ip)
       Host.set_names(ip, [])
     end
 
+    # Exports udev rules for AY profile
+    def export_udevs(devices)
```

Comment:
my hurts pain when seeing this method. I expect it is just copy and paste, but I think at least some love it deserve.